content/pt-br/docs/client-options.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Implementações de Clientes ACME
+slug: client-options
+top_graphic: 1
+lastmod: 2019-05-24
+---
+
+{{< clientslastmod >}}
+
+A Let's Encrypt usa o protocolo ACME para verificar que você controla dado nome
+de domínio e para lhe emitir um certificado. Para obter um certificado Let's Encrypt, você
+precisará escolher um cliente ACME para usar.
+
+Os clientes ACME abaixo são oferecidos por terceiros. A Let's Encrypt não controla ou revisa
+clientes de terceiras e não pode fazer garantias a respeito de sua segurança ou confiabilidade.
+
+Alguns clientes ACME que rodam diretamente no navegador estão disponíveis, mas não os listamos aqui porque
+eles encorajam um processo de renovação manual que resulta em uma experiência de uso ruim 
+e aumenta o risco do usuário esquecer de renovar um certificado. 
+
+# Recomendado: Certbot
+
+Recomendamos que os usuários comecem pelo cliente [Certbot](https://certbot.eff.org/). Ele pode desde apenas obter o certificado para você até mesmo instalá-lo, dependendo do que você preferir. Ele é fácil de usar, funciona em muitos sistemas operacionais e possui uma ótima documentação.
+
+Se o certbot não supre suas necessidades ou você quer testar algo diferente, existem muitos outros clientes
+que você pode usar na lista abaixo, agrupados por linguagem de programação e ambiente em que são executados.
+
+{{< clients acme_v2="ACME v2 Compatible Clients" libraries="Libraries" projects="Projects integrating with Let’s Encrypt" >}}
```

Comment:
"ACME v2 Compatible Clients", "Libraries" and "Projects integrating with Let’s Encrypt" can be translated

Reply:
Oops I didn't know these were parameters to be used on the rendered page, it has been fixed